iFixit/react-commerce
# Remove featured product list section from product list page - backend changes

🛑 deploy_block on #1488

This PR aim at cleaning up Strapi after the removal of the featured-product-list section.

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley @masonmcelvain

### QA

1. Visit Vercel preview
2. Verify that product list pages are still working correctly

### Before merge

To be sure not having any sort of downtime and allowing Strapi to restart smoothly, we should take care that the production data do not include any featured-product-list section at all.
The fastest way I can think of is seeding local environment with production data and checking in the sqlite db that the `product_list_components` collection is not referencing any featured-product-list section at all.
This should be performed just before merge to be sure no section of this kind is present, not even due to testing or by chance.

To do so you have to:

1. On your local machine, make sure that the `backend` folder `.env` includes the following environment variable: `STRAPI_ADMIN_ENABLE_ADDONS_DANGEROUS_ACTIONS=true`
2. Start the dev environment and navigate to `http://localhost:1337/admin/plugins/addons`
3. You should see an interface with an input field - if not try to delete your `.cache`, `.tmp` and `.dist` folders from your local `backend` folder and restart the dev environment.
4. Input https://strapi.ifixit.com/ and press `Reset feed` to overwrite your local data with production data
5. Connect to the local sqlite instance located in `{project-folder}/backend/.tmp/data.db`
6. Inspect the `product_list_components` table to check for the presence of any featured-product-list
